spec/crypto_env_var/cipher_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+
+RSpec.describe CryptoEnvVar::Cipher do
+  let(:plaintext) do
+    <<-EOS
+    How much wood could a woodchuck chuck 
+    If a woodchuck could chuck wood? 
+    As much wood as a woodchuck could chuck, 
+    If a woodchuck could chuck wood.
+    EOS
+  end
+
+  describe "a Cipher initialized with a private key" do
+    subject { described_class.new(private_key) }
+
+    it "can encrypt and decrypt a text" do
+      ciphertext = subject.encrypt(plaintext)
+      expect(ciphertext).to_not eq plaintext
```

Comment:
Nit - a rule I know is one expectation per test case (`it`).

Reply:
I know where you're coming from, but I started to appreciate the "it depends" stance.

In this case I'm trying to tell a story and describe the behaviour, and both expectations are required to convey the single concept tested in the example. Does it make sense?

Reply:
Sure. I'm not religious about it.